### PR TITLE
Create scripts in .ci dir to install CP/DP

### PR DESCRIPTION
**Pull request by micpez**
Move install step logic from groovy file to bash scripts in .ci
directory in ngic-rtc repo.

### CODE REVIEW

**Comment by micpez** [comment]
Needs to be merged together with https://github.com/omec-project/omec-project-ci/pull/41

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Review by krsna1729 — COMMENTED** [review]
Can we shrink the license header to match rest of the files in the repo?

```
#!/bin/bash

# SPDX-License-Identifier: Apache-2.0
# Copyright 2019-present Open Networking Foundation
```

**Comment by micpez** [comment]
> # SPDX-License-Identifier: Apache-2.0
> # Copyright 2019-present Open Networking Foundation

@zdw is this OK?

**Comment by zdw** [comment]
> > `# SPDX-License-Identifier: Apache-2.0`
> > `# Copyright 2019-present Open Networking Foundation`
> 
> @zdw is this OK?

I'll ask, and let you know when I get a response.

**Comment by zdw** [comment]
@micpez @krsna1729 The shortened copyright header should be fine.

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
retest this please

**Comment by micpez** [comment]
Done

File: .ci/common/utils.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+get_sgx_dealer_keys()
+{
+    # Fetch keys from SGX-dealer.
+
+    local INSTALL_PATH=$1
+    local KEYS=
+    # shellcheck disable=SC2029
+    KEYS=$(ssh sgx-kms-cdr "source ${INSTALL_PATH}/c3po/.ci/scripts/utils.sh; get_sgx_dealer_keys ${INSTALL_PATH}")
```
Comment (krsna1729) [inline .ci/common/utils.sh]: This seems like inception. please explain whats happening here.

Comment (krsna1729) [reply .ci/common/utils.sh]: never mind got it. c3po vs ngic-rtc

**Review by krsna1729 — APPROVED** [review]

**Comment by zdw** [comment]
Very stale, no longer relevant?

**Pull request closed without merging**